packages/core/src/user-store/postgres/store.ts

```diff
@@ -538,15 +538,21 @@ export class PostgresUserStore implements UserStore {
       effectiveTo: MAX_INTEGER,
     }));
 
-    const instances = await this.db
-      .insertInto(tableName)
-      .values(createInstances)
-      .returningAll()
-      .execute();
+    const chunkSize = 1000;
```

Comment:
I would probably call this `MAX_BATCH_SIZE` and move it to the top of the file alongside `MAX_INTEGER`. Same goes for the SQLite store.